tests/integration/test_model.py

```diff
@@ -58,6 +67,60 @@ def test_training_and_prediction(c, training_df):
     check_trained_model(c)
 
 
+def test_cuml_training_and_prediction(c, gpu_training_df):
+    cuml = pytest.importorskip("cuml", reason="cuml not installed")
+    model_query = """
+        CREATE OR REPLACE MODEL my_model WITH (
+            model_class = 'cuml.linear_model.LogisticRegression',
+            wrap_predict = True,
+            wrap_fit = False,
+            target_column = 'target'
+        ) AS (
+            SELECT x, y, x*y > 0 AS target
+            FROM timeseries
+        )
+        """
+    c.sql(model_query)
+    check_trained_model(c)
+
+
+def test_dask_cuml_training_and_prediction(c, gpu_training_df, client):
+    cuml = pytest.importorskip("cuml", reason="cuml not installed")
+
+    model_query = """
+        CREATE OR REPLACE MODEL my_model WITH (
+            model_class = 'cuml.dask.linear_model.LinearRegression',
+            target_column = 'target'
+        ) AS (
```

Comment:
Error Trace: 

```python

distributed.worker - WARNING - Compute Failed
Function:  _get_model_attr
args:      (LinearRegressionMG(), '__deepcopy__')
kwargs:    {}
Exception: 'AttributeError("Attribute __deepcopy__ does not exist on model <class \'cuml.linear_model.linear_regression_mg.LinearRegressionMG\'>")'
```

Probably due to creating a duplicate context at [line](https://github.com/dask-contrib/dask-sql/blob/ebdf4d5e5e5d622bfda881129620a4216f16e453/dask_sql/physical/rel/custom/predict.py#L91-L92) here .

Reply:
Raised https://github.com/dask-contrib/dask-sql/issues/332 to track this issue.